### DISCUSSION TITLE
We need a new release but...

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#2812] [Questions & Answers] [answered]
Hi people,
our current release has a typo in a function name that's creating problems to our users, the bug was already fixed but we need to release.

Problem is, we can't release until https://github.com/OpenMage/magento-lts/pull/2165 is merged because, since the introduction of composer for some of the 3rd party code we had in our repo, we've to build a "release zip file" for people using openmage "the old way".

So I'm asking for reviews of that PR, which I think should already be ready to go.
Thanks everybody!

**Comment by Flyingmana** [comment]
Problem is resolved?

**Comment by sreichel** [comment]
Release-builder works, but we have some issues. Vote for `-RC1` release.

**Comment by fballiano** [comment, marked as answer]
This specific question is not relevant anymore
